src/main/java/com/geronimo/model/User.java

```diff
@@ -46,6 +55,7 @@ private void removeFollower(User follower) {
     public void addMessage(Message message) {
         if (message != null) {
             messages.add(message);
+            messages.add(message);
```

Comment:
is it to make sure that if user wasn't added at the first attempt it would be added at the second? :)

Reply:
Accident